docs/advanced-topics/messaging-system.md

```diff
@@ -1,230 +1,16 @@
 ---
-id: messaging-system
-title: Messaging System
-sidebar_label: Messaging System
+id: introduction
+title: Introduction
+sidebar_label: Introduction
```

Comment:
We can't keep using Introduction for content. All titles need to be specific to their section. If we have multiple titles that are the same, we'll have search problems. Changing to About Messaging System

Reply:
I get the issue it was because i didn't want to have the same name as the section. I am not sure about it being about messaging system as that might imply that thats everything in a single page.  perhaps I should change it to introduction to messaging system

Reply:
Updated to 'Introduction to the messaging system'